### ISSUE TITLE
About this Project

### ISSUE THREAD

**Issue by rajeevrra** [k8-proxy/p-k8-tech-radar#2]
This project is to create a ready reference technology landscape for Glasswall solutions inhouse systems.
Based on the reference link:
https://www.thoughtworks.com/radar/techniques
We need to map the given Tech stack appropriately proper labels, technology etc.
This is a Priority 2 (P2) project.

**Comment by rajeevrra** [comment]
Hi @dinis@dtollaku : Greetings

Further to what I know above about the Tech Radar project(based on my discussion with @Dinis), I would like to know the following information as well. This is also essential for Steve's Project - Wardley Maps. I need to be ready with this by tomorrow EOD. Looking forward for your help and support. :

 . Who are your Users?
2. What are their high-level needs?
3. At a high-level, how do you meet these needs? (eg: what are your high-level capabilities?)
4. What sorts of products/services do you need?
5. Do you rely on the products/services from another Glasswall OS team?
6. What management methods are you using?
In addition, if there is anything else that can be added to the description that I have put above or any correction in my understanding, kindly let me know.
Please let me know if this needs to be routed to someone else
Looking forward for your help and support. Above answers will help us deliver the project better.

**Comment by rajeevrra** [comment]
Key FAQ's(Frequently answered questions):

Who are your Users?
All Developers of Glasswall Solutions
2. What are their high-level needs?
To have a ready reckoner technology stack artefact as to ascertain any new project feasibility.
3. At a high-level, how do you meet these needs? (eg: what are your high-level capabilities?)
By having a quick reference artefact like the Tech Radar file.
4.What sorts of products/services do you need?
Basically a simple Info Graphics File which contains the ready reckoner Technology file.
5. Do you rely on the products/services from another Glasswall OS team?
Yes, inputs for this project will be taken from other Glasswall team as we build this.
What management methods are you using?
Simple Agile methodology
